test/functionalTest/cases/1280_patch_erroneous_service_path/patch_erroneous_service_path.test

```diff
@@ -0,0 +1,123 @@
+# Copyright 2015 Telefonica Investigacion y Desarrollo, S.A.U
+#
+# This file is part of Orion Context Broker.
+#
+# Orion Context Broker is free software: you can redistribute it and/or
+# modify it under the terms of the GNU Affero General Public License as
+# published by the Free Software Foundation, either version 3 of the
+# License, or (at your option) any later version.
+#
+# Orion Context Broker is distributed in the hope that it will be useful,
+# but WITHOUT ANY WARRANTY; without even the implied warranty of
+# MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE. See the GNU Affero
+# General Public License for more details.
+#
+# You should have received a copy of the GNU Affero General Public License
+# along with Orion Context Broker. If not, see http://www.gnu.org/licenses/.
+#
+# For those usages not covered by this license please contact with
+# iot_support at tid dot es
+
+# VALGRIND_READY - to mark the test ready for valgrindTestSuite.sh
+
+--NAME--
+Patch Erroneous Service Patch 
+
+--SHELL-INIT--
+dbInit CB
+brokerStart CB
+
+--SHELL--
+
+#
+## 01. PATCH /v2/ent=ities/E1 (invalid URI)
+## 02. PATCH /v2/entities/E1 (incorrect service path)
+## 03. PATCH //v2/entities/E1 (service path without /) 
+## 04. PATCH //v2/entities/E1 (unrecognized content type)
+
```

Comment:
What it important is to have the comment up here reflecting exactly what the test does.
**//v2/entities/E1** would probably give an error ...
The real test doesn't have the "double-slash" ...